-------------------------------------------------------------------------------
Test set: BaseTest
-------------------------------------------------------------------------------
Tests run: 7, Failures: 0, Errors: 7, Skipped: 0, Time elapsed: 11.829 s <<< FAILURE! - in BaseTest
test4_CreateThenDeleteFixtureAndCheckRemoval(BaseTest)  Time elapsed: 11.828 s  <<< ERROR!
java.net.ConnectException: Connection refused: connect
	at BaseTest.createFixture(BaseTest.java:34)
	at BaseTest.lambda$test4_CreateThenDeleteFixtureAndCheckRemoval$2(BaseTest.java:93)

test3_CreateFixtureAndRetrieveAsSoonAsAvailable(BaseTest)  Time elapsed: 11.828 s  <<< ERROR!
java.net.ConnectException: Connection refused: connect
	at BaseTest.createFixture(BaseTest.java:34)
	at BaseTest.lambda$test3_CreateFixtureAndRetrieveAsSoonAsAvailable$1(BaseTest.java:82)

test1_GetAllFixtures(BaseTest)  Time elapsed: 7.741 s  <<< ERROR!
java.net.ConnectException: Connection refused: connect
	at BaseTest.test1_GetAllFixtures(BaseTest.java:57)

test2_CreateNewFixtureAndAssertTeamId(BaseTest)  Time elapsed: 11.828 s  <<< ERROR!
java.net.ConnectException: Connection refused: connect
	at BaseTest.createFixture(BaseTest.java:34)
	at BaseTest.lambda$test2_CreateNewFixtureAndAssertTeamId$0(BaseTest.java:70)

test4_CreateThenDeleteFixtureAndCheckRemoval(BaseTest)  Time elapsed: 0 s  <<< ERROR!
java.net.ConnectException: Connection refused: connect
	at BaseTest.tearDown(BaseTest.java:50)

test2_CreateNewFixtureAndAssertTeamId(BaseTest)  Time elapsed: 0 s  <<< ERROR!
java.net.ConnectException: Connection refused: connect
	at BaseTest.tearDown(BaseTest.java:50)

test3_CreateFixtureAndRetrieveAsSoonAsAvailable(BaseTest)  Time elapsed: 0 s  <<< ERROR!
java.net.ConnectException: Connection refused: connect
	at BaseTest.tearDown(BaseTest.java:50)

